pandas_dev_flaker/_ast_helpers.py

```diff
@@ -30,3 +31,19 @@ def check_for_wrong_alias(
             return name_.asname != alias
     else:
         return False
+
+
+def is_str_constant(
+    node: ast.Call,
+) -> bool:
+    python_version = float(sys.version_info.major) + 0.1 * float(
+        sys.version_info.minor,
+    )
```

Comment:
😄  never do this, just use `sys.version_info`

Reply:
besides, ask yourself what would happen for `Python 3.10`...

Reply:
Sorry :smile:
I'm gonna fix that!